vite.config.ts

```diff
@@ -4,4 +4,24 @@ import react from "@vitejs/plugin-react";
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
+  build: {
+    rollupOptions: {
+      input: {
+        main: "index.html", // The entry point for your React app
+        content: "src/content/content.js", // The entry point for your content script
+      },
+      output: {
+        entryFileNames: (chunkInfo) => {
+          if (chunkInfo.name === "content") {
+            return "content.js";
+          }
+          // For all other entries, retain the default behavior which includes hashing and assets folder
+          return "assets/[name]-[hash].js";
+        },
+        chunkFileNames: "assets/[name]-[hash].js",
+        assetFileNames: "assets/[name]-[hash].[ext]",
+      },
+    },
+  },
+  assetsInclude: ["src/content/content.css"],
```

Comment:
A bit much and may want to reconfigure, but this is a way to place the content.js in the dist folder upon building